ci-automation/garbage_collect.sh

```diff
@@ -92,6 +92,7 @@ function garbage_collect() {
             rmpat="${rmpat} ${BUILDCACHE_PATH_PREFIX}/boards/*/${os_vernum}/"
             rmpat="${rmpat} ${BUILDCACHE_PATH_PREFIX}/containers/${os_docker_vernum}/flatcar-images-*"
             rmpat="${rmpat} ${BUILDCACHE_PATH_PREFIX}/images/*/${os_vernum}/"
+            rmpat="${rmpat} ${BUILDCACHE_PATH_PREFIX}/testing/${os_vernum}/"
```

Comment:
Should this also be `testing/*/${os_vernum}`?

Reply:
No - for compatibility reasons we store packages and images at <arch>/<version> (like in the google bucket) - but this makes it hard to browse as Packages and images of the same version are scattered across 4 directories.
For the testing artifacts I've changed that, so everything belonging to one version is below the same version directory.
This corresponds to https://github.com/flatcar-linux/scripts/blob/3397167b5e90d155cf3f64d00b4d4e40efbf2121/ci-automation/test.sh#L180-L183 (`... "testing/${vernum}/${arch}/${image}" ...`).